### ISSUE TITLE
Logging drivers to be able to deliver log messages as is

### ISSUE THREAD

**Issue by LakshithadSilva** [docker/docker#24898]
Docker logging drivers is a fantastic feature. However, I would like some logging drivers (such as the Splunk driver) to deliver log messages as is, i.e. without wrapping them in a JSON object. This indeed means we will lose the extra metadata like tags and labels. The issue with wrapping the raw log message is that we need to reconfigure the recipient to unwrap the message. This can be inefficient with high volume situations and adds complexity for DevOps. A "--log-opt" option to disable logging metadata would be useful in my opinion.

**Comment by thaJeztah** [comment]
This looks like a duplicate of https://github.com/docker/docker/issues/17020, which was rejected, so unfortunately, I'll have to close this one, but feel free to comment after I closed

**Comment by LakshithadSilva** [comment]
I am afraid you haven't really understood the scenario I described. I have seen that other issue which was closed, but what I proposed is not the same.

As I mentioned, I would like a logging driver such as the Splunk driver to NOT wrap the raw message within a JSON object. I am not interested in using a plain text driver such as syslog because it won't be forwarding messages to Splunk.

For example, a log message forwarded to Splunk by the Splunk logging driver looks similar to this:

`{"line":"2016.07.21 15:23:57 samples.xmpp.ClientSession - Session created for user [abcd]","source":"stdout","tag":"443f6ddafd36"}`

Instead of the above, I would like the Splunk driver to forward the raw log message:

`2016.07.21 15:23:57 samples.xmpp.ClientSession - Session created for user [abcd]`

I hope this explanation helps for this requested feature to re-evaluated.

Thank you.
